internal/services/openai.go

```diff
@@ -69,12 +73,25 @@ func (o openAI) askChatGPT(body io.Reader) (*ChatGPTResponse, error) {
 	req.Header.Set("Content-Type", "application/json")
 	req.Header.Set("Authorization", "Bearer "+o.token)
 
+	var currentReqResponseTime time.Duration
+	trace := &httptrace.ClientTrace{
+		GotFirstResponseByte: func() {
```

Comment:
I wonder if first byte is the right thing. Is first byte-to-last byte pretty fast?

Reply:
Do I care about first byte or last byte? Anyway, this is fine.